.github/workflows/release-upload.yml

```diff
@@ -159,10 +159,10 @@ jobs:
               derivation+="${{ matrix.arch }}"
               ;;
             "x86_64-linux")
-              derivation+="x86_64-linux.ghc965-x86_64-unknown-linux-musl"
+              derivation+="x86_64-linux.ghc96-x86_64-unknown-linux-musl"
               ;;
             "aarch64-linux")
-              derivation+="x86_64-linux.ghc965-aarch64-unknown-linux-musl"
+              derivation+="x86_64-linux.ghc96-aarch64-unknown-linux-musl"
```

Comment:
* [ ] double check derivation name

Reply:
@carbolymer> please try the release pipeline on this branch as follows:

```sh
gh workflow run "Release Upload" -r $(git branch --show-current)
```

Because the HEAD of this branch has no associated tag, it will run the pipeline but not create a release, so it's a good way to test your change. See the documentation: https://github.com/IntersectMBO/cardano-cli/blob/master/.github/workflows/release-upload.yml#L28

Reply:
Seems that without patch name it's not working. Switched to 966 in the other PR.